r/restic/restic_ubi8.4.sh

```diff
@@ -0,0 +1,85 @@
+# ----------------------------------------------------------------------------
+#
+# Package       : restic
+# Version       : v0.12.1
+# Source repo   : https://github.com/restic/restic/
+# Tested on     : rhel UBI 8.4
+# Script License: Apache License, Version 2 or later
+# Maintainer's  : Santosh Magdum <santosh.magdum@us.ibm.com>
```

Comment:
Please use your name and email ID in the maintainer's field

Reply:
Done.